modules/service/src/main/scala/lucuma/odb/service/GuideService.scala

```diff
@@ -962,7 +948,6 @@ object GuideService {
           obs.c_hour_angle_max,
           obs.c_pac_mode,
           obs.c_pac_angle,
-          obs.c_spec_wavelength,
```

Comment:
Should we drop this column?

Reply:
We don't use it for anything, so it seems like we should. Is there a reason not to?

Reply:
I have no idea why I originally added the `obs` table alias since there are no joins. I'll get rid of it.

Reply:
Do we store there the requirement wavelength?

Reply:
Yes

Reply:
Oh, did you mean actually drop the column from the t_observation table? Not just the query. Yeah, that is the spectroscopy requirements wavelength value.

Reply:
Raul did talk to Andy about dropping either the spec requirements `wavelength` or the `wavelengthAt` value, since they don't both seem necessary. Andy said "_we'll see_", or something to that effect.